webserver/src/main/java/io/kestra/webserver/controllers/api/LogController.java

```diff
@@ -43,6 +44,9 @@ public class LogController {
     @Inject
     private TenantService tenantService;
 
+    @Inject
+    private LogService a;
```

Comment:
This is not acceptable

Reply:
what should i  name it then?

Reply:
loginService named already exist with : 
![image](https://github.com/user-attachments/assets/c665f85c-ed04-46c9-94b3-b584485c0934)

Reply:
the current `logService` should be renamed `executionLogService`, so your `logService` that use the class `LogService` can keep his name, so we know  which class is used behind the variable name